semantic-focus-a20046.md

```diff
@@ -0,0 +1,120 @@
+---
+id: a20046
+name: Sequential focus has semantic role
+rule_type: atomic
+
+description: | 
+ This rule checks that every element that is part of the sequential focus navigation order has a semantic role.
+
+accessibility_requirements: 
+  wcag20:4.1.2: # Name, Role, Value (A)
```

Comment:
I would (still) not say this fails WCAG 4.1.2. 

A Div with tabindex is not a user interface component and doesn't need a role.